### ISSUE TITLE
Call to a member function toDateTimeString() on string

### ISSUE THREAD

**Issue by acoustep** [RoumenDamianoff/laravel-feed#39]
Hi, I have an issue using 2.8.3 in a Laravel 4 application

I have looked through the source and when using the carbon format and rending as "rss", ```formatDate``` is called on the feed items twice.

Because ```formatDate``` turns a carbon object into a string, the second time is causing this error:

```
Call to a member function toDateTimeString() on string
```

The first time it is called in the add method on line 69 of Feed.php

```
$pubdate = $this->formatDate($pubdate);
```

The second time it is called during the render method in the  $format == 'rss' statement on line 154

```
$this->items[$k]['pubdate'] = $this->formatDate($this->items[$k]['pubdate'], "rss");
```

I'm currently working around this by using ```datetime``` format instead, but I thought i'd let you know of the issue.

**Comment by RoumenDamianoff** [comment]
Hi, this bug is fixed in v2.8.5 for laravel 4

**Comment by acoustep** [comment]
Thanks!
